_data/faq.yml

```diff
@@ -458,46 +458,59 @@
 
       - id: data-on-sd
         en:
-            title: c:geo data is always stored in internal memory!
+            title: How can I store c:geo data on my SD card?
             content: |
                 There are two things we are storing on your device:
```

Comment:
IMHO not here. I left them out in the first version of the FAQ (which I adapted here) as those are special files the user creates on demand and this entry is dealing with solution for running short on disk space.